extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -130,6 +131,17 @@ export class TargetSelectionPage extends MigrationWizardPage {
 
 	public async onPageLeave(pageChangeInfo: azdata.window.WizardPageChangeInfo): Promise<void> {
 		this.wizard.registerNavigationValidator((e) => {
+
+			sendSqlMigrationActionEvent(
+				TelemetryViews.MigrationWizardTargetSelectionPage,
+				TelemetryAction.OnPageLeave,
+				{
+					'sessionId': this.migrationStateModel?._sessionId,
+					'subscriptionId': this.migrationStateModel?._targetSubscription?.id,
+					'resourceGroup': this.migrationStateModel?._resourceGroup?.name,
+					'tenantId': this.migrationStateModel?._accountTenants[0]?.id
```

Comment:
Is grabbing the first one guaranteed to be correct?

Reply:
@rachel-msft  can you pls confirm? this is how we were doing before.

Reply:
@rchlkm